common/beerocks/bcl/source/beerocks_ucc_listener.cpp

```diff
@@ -563,22 +563,14 @@ void beerocks_ucc_listener::handle_wfa_ca_command(const std::string &command)
             LOG(ERROR) << "failed to send reply";
             break;
         }
-        auto parameter = params["parameter"];
-        std::transform(parameter.begin(), parameter.end(), parameter.begin(), ::tolower);
         std::string value;
-        if (parameter == "alid") {
-            if (!net::network_utils::linux_iface_get_mac("br-lan", value)) {
-                LOG(ERROR) << "failed to get br-lan mac address";
-                reply_ucc(eWfaCaStatus::ERROR, "failed to get br-lan mac address");
-                break;
-            }
-        } else if (!handle_dev_get_param(params, value)) {
-            LOG(ERROR) << "failed to get parameter " << parameter << "error: " << value;
+        if (!handle_dev_get_param(params, value)) {
+            LOG(ERROR) << "failed to get parameter " << params["parameter"] << "error: " << value;
             reply_ucc(eWfaCaStatus::ERROR, value);
```

Comment:
I don't understand this comment - we check that `params["parameter"]` exists in `parse_params` in line 557 where "parameter" is a mandatory param, so its fine to access it with `[]` operator.
Regarding your other comment - since we pass the entire parameters map to the derived class it doesn't make sense to do any conversions in the base class - some parameters might require to be upper case, and we can't assume otherwise in the base `dev_get_parameter` implementation which does nothing but call the derived implmenetation.